### DISCUSSION TITLE
Using Intents to turn off the WiFi

### DISCUSSION THREAD

**Discussion by LarchyEverything** [Nain57/Smart-AutoClicker#524] [Q&A]
I am not the most savvy but I need to be able to turn on and off the wifi. I have found intents online but none of them properly fit into the sacs intents and don't work either. I'm hoping I can get some help because I haven't found anything anywhere else.

**Comment by hsantanna** [comment]
Can't you disable / enable wifi by clicking?

**Reply by LarchyEverything** [reply]
No. Not all phones let accessibility things even access settings or the phone UI.
